ZuluContent/Mobiles/PlayerMobile/PlayerMobile.cs

```diff
@@ -715,6 +723,8 @@ public override void OnItemAdded(Item item)
             if (NetState != null)
                 CheckLightLevels(false);
 
+            OutgoingZuluPackets.SendZuluPlayerStatus(NetState, this);
```

Comment:
Wondering - will some of the buffs that add prots (eg Antidote poison prot) need to send this as well?

Reply:
Yeah good point, need to figure out all the situations where the packet needs to be sent